common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -117,7 +119,7 @@ trait Logging {
     }
   }
 
-  private def withLogContext(context: java.util.HashMap[String, String])(body: => Unit): Unit = {
+  protected def withLogContext(context: java.util.HashMap[String, String])(body: => Unit): Unit = {
```

Comment:
Why change `private` to `protected`
because some class extends `Logging` and override the method `logInfo`, `logWarn`, 'logError', eg: